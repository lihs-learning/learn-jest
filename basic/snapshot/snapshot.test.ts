import {
  getConfigWithoutTime,
  getConfigWithTime,
} from "./snapshot"

describe('快照', () => {
  test('getConfigWithoutTime', () => {
    expect(getConfigWithoutTime()).toMatchSnapshot()
  })

  test('getConfigWithTime', () => {
    expect(getConfigWithTime()).toMatchSnapshot({
      time: expect.any(Date),
    })
  })
})
