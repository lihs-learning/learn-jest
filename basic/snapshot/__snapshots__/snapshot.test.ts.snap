// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`快照 getConfigWithoutTime 1`] = `
Object {
  "port": 0,
  "server": "",
}
`;
